src/mlpack/core/data/dataset_mapper_impl.hpp

```diff
@@ -21,17 +21,16 @@ namespace data {
 
 // Default constructor.
 template<typename PolicyType>
-inline DatasetMapper<PolicyType>::DatasetMapper(const size_t dimensionality) :
-    types(dimensionality, Datatype::numeric)
+inline DatasetMapper<PolicyType>::DatasetMapper(const size_t dimensionality)
+    : types(dimensionality, Datatype::numeric)
```

Comment:
Everything in mlpack has the `:` on the preceding line so that the initialization lists can line up at four-space indentation, so we should move that back to how it was.